Makefile

```diff
@@ -349,6 +349,7 @@ tests/%.parse: tests/%
 tests/specs/functional/%.prove:                 TEST_SYMBOLIC_BACKEND = haskell
 tests/specs/examples/%.prove:                   TEST_SYMBOLIC_BACKEND = haskell
 tests/specs/erc20/hkg/totalSupply-spec.k.prove: TEST_SYMBOLIC_BACKEND = haskell
+tests/specs/benchmarks/address00-spec.k.prove:  TEST_SYMBOLIC_BACKEND = haskell
```

Comment:
@ehildenb I'd like to run these spec proof tests for both java and haskell backends. Could you please reorg  Makefile to do that? I think we want to do:
- run all proof tests with the java backend
- _dry-run_ all proof tests with the haskell backend (this can be dropped once the porting is done)
- run all _ported_ proof tests with the haskell backend

Reply:
I would rather not re-org the makefile to do this, we used to do this but I don't really care to support the Java backend with ported proofs.

Reply:
But if you absolutely want to, it's better to do it in the Jenkinsfile. We can have separate stages for running all the proofs with haskell vs. java (the ones that are supported).

Reply:
OK. I'll think about it. Let's focus on merging this PR then, and later let me open another PR if it is really needed.

Reply:
I'll have a PR ready in half an hour or so that enables running them on both.